include/boost/json/parse_options.hpp

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (c) 2020 Krystian Stasiowski (sdkrystian@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/cppalliance/json
+//
+
+#ifndef BOOST_JSON_PARSE_OPTIONS_HPP
+#define BOOST_JSON_PARSE_OPTIONS_HPP
+
+/** Parser Extensions.
```

Comment:
There will eventually need to be more written here

Reply:
You could say "the default parse options allow only strict JSON."

Reply:
I've added a bit.